packages/devtools_app/lib/src/common_widgets.dart

```diff
@@ -988,3 +989,20 @@ class _BreadcrumbPainter extends CustomPainter {
         colorScheme != oldDelegate.colorScheme;
   }
 }
+
+class FormattedJson extends StatelessWidget {
+  const FormattedJson({@required this.json});
+
+  static const encoder = JsonEncoder.withIndent('  ');
+
+  final Map<String, dynamic> json;
+
+  @override
+  Widget build(BuildContext context) {
+    final formattedArgs = encoder.convert(json);
+    return Text(
+      formattedArgs.replaceAll('"', ''),
+      style: const TextStyle(fontFamily: 'RobotoMono'),
```

Comment:
don't we already reference the monospace font somewhere in our theme?

Reply:
we have `fixedFontStyle`. Used that.